packages/cyberstorm/src/components/componentStyles/PackageCard.module.css

```diff
@@ -0,0 +1,130 @@
+/* Common PackageCard styles */
+.root {
+  display: flex;
+  flex: none;
+  flex-direction: column;
+  gap: var(--gap-m);
+  align-items: flex-start;
+
+  border-radius: var(--border-radius-m);
+  overflow: hidden;
+}
+
+.root:hover {
+  box-shadow: var(--box-shadow-1);
+}
+
+.imageWrapper {
+  position: relative;
+  width: 100%;
+  padding-bottom: 100%;
+  overflow: hidden;
+  transition: 0.2s;
+}
+
+.root:hover .imageWrapper {
+  filter: brightness(1.15);
+}
+
+/* Styles for PackageCard's image */
+.image {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 100%;
+  height: auto;
+  transform: translate(-50%, -50%);
```

Comment:
`translate(-50%, -50%)` could declared as a local variable so it can be referenced in the `:hover` below instead of typing in twice.

Reply:
👍